packages/packages/commonwell-sdk/src/client/commonwell.ts

```diff
@@ -418,6 +420,22 @@ export class CommonWell {
     return personSearchRespSchema.parse(resp.data);
   }
 
+  /**
+   * Gets a person based on person id.
+   * See: https://specification.commonwellalliance.org/services/patient-identity-and-linking/protocol-operations#8713-find-persons-matching-patient-demographics
+   *
+   * @param meta        Metadata about the request.
+   * @param personId   The person ID.
+   * @returns
+   */
+  async searchPersonByUri(meta: RequestMetadata, personId: string): Promise<Person> {
```

Comment:
Isn't this a "get"? It would be nice to name accordingly. Example: `getPatientLink()`